### PR TITLE
Fix documentation and license file omissions

### PR DESCRIPTION
**Pull request by irm-codebase**
This PR fixes some omissions seen in the PR of the transport module, and ensures sequential attribution of the software.

### CODE REVIEW

File: modules/hydropower/LICENSE [hunk 1]
```diff
@@ -0,0 +1,22 @@
+MIT License
+
+Copyright (c) 2018-2021 Tim Tröndle (ETH Zürich / IASS Potsdam), Bryn Pickering (ETH Zürich)
+Copyright (c) 2024 AUTHORS
```
Comment (irm-codebase) [inline modules/hydropower/LICENSE:4]: If the module was taken from EC, attribution is given to the original authors, and then to the authors at `ec_modules`.

File: modules/transport_road/LICENSE [hunk 2]
```diff
@@ -1,6 +1,8 @@
 MIT License
 
-Copyright (c) 2021, Francesco Sanvito (TU Delft), Bryn Pickering (University of Cambridge), Adrien Mellot (ETH Zürich), Ivan Ruiz Manuel (TU Delft)
+Copyright (c) 2021 Bryn Pickering (ETH Zürich), Tim Tröndle (ETH Zürich / IASS Potsdam)
+Copyright (c) 2024 Adrien Mellot (ETH Zürich)
+Copyright (c) 2024 AUTHORS
```
Comment (irm-codebase) [inline modules/transport_road/LICENSE:5]: In this case, Adrien is included since he contributed during the sprint.

**Review by irm-codebase — COMMENTED** [review]
The most important fixes are the attribution updates.

The rest are some minor documentation, which will improve as the modules mature.

File: modules/hydropower/LICENSE (hunk 1 above)
Comment (brynpickering) [reply modules/hydropower/LICENSE:4]: `-2022` and does that assume 2022-2024 is copyright-free?

File: modules/transport_road/LICENSE (hunk 2 above)
Comment (brynpickering) [reply modules/transport_road/LICENSE:5]: Would remove Tim from the initial copyright

File: LICENSE [hunk 3]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2024 Calliope
+Copyright (c) 2024 AUTHORS
```
Comment (brynpickering) [inline LICENSE:3]: In calliope we do `contributors listed in AUTHORS.` to make it clearer. Don't mind either way.

File: modules/hydropower/LICENSE (hunk 1 above)
Comment (irm-codebase) [reply modules/hydropower/LICENSE:4]: I'll ask the license expert about this tomorrow. I assume it does not really matter.
Still, I've added those years.

File: LICENSE (hunk 3 above)
Comment (irm-codebase) [reply LICENSE:3]: I'll leave this as-is for now. Later we might move to a CITATION and LICENSE method, depending on what the expert says.

**Comment by irm-codebase** [comment]
@brynpickering updated the license information with your feedback. I've also merged `heat` with better co-author attribution (you should be able to see this in the main repo).

**Comment by irm-codebase** [comment]
@brynpickering is this OK to merge?

**Review by brynpickering — APPROVED** [review]

**Pull request merged**